Update icons handling (#657)
smile-io/ember-polaris: docs/icon.md

## Icon

`PolarisIcon` implements the [Polaris Icon component](https://polaris.shopify.com/components/images-and-icons/icon).

### Examples

Basic usage:

```hbs
<PolarisIcon @source="NotesMajor" />
```

Customizing with color and backdrop:

```hbs
<PolarisIcon @source="AddMinor" @color="tealDark" @backdrop={{true}} />
```

### Rendering custom icons

By default, `PolarisIcon` will render SVG's passed directly as strings to `@source` as external images.

> If you want to render custom icons similarly as main Polaris icons are rendered, you can configure `ember-svg-jar` in your app instead.

Reviewer: Possible slight tweak for grammar:

```suggestion
If you want to render custom icons in the same way that the built-in Polaris icons are rendered, you can configure `ember-svg-jar` in your app instead.
```
